jxl/src/frame/modular/decode.rs

```diff
@@ -102,18 +89,31 @@ fn decode_modular_channel(
     Ok(())
 }
 
-pub fn decode_modular_section(
-    buffers: &mut [ModularBufferInfo],
-    buffer_indices: &[usize],
+// This function will decode a header and apply local transforms if a header is not given.
+// The intended use of passing a header is for the DcGlobal section.
+pub fn decode_modular_subbitstream(
+    buffers: &mut [impl DerefMut<Target = Image<i32>>],
+    channel_indices: &[usize],
     grid_index: usize,
     stream_id: usize,
-    header: &GroupHeader,
+    header: Option<GroupHeader>,
```

Comment:
(Not &?)

Reply:
No, & makes things more annoying for the `if` and doesn't actually help callers...